manage.yml

```diff
@@ -0,0 +1,35 @@
+project_name: Robottelo
+help_text: This is the {project_name} interactive shell
+
+# interactive shell
+shell:
+  auto_import:
+    display: true
+    objects:
+        robottelo.config.settings:
+          as: settings
+          init: configure
+        nailgun.entities:
+          as: entities
+        nailgun:
+          as: ng
+        robottelo:
+          as: rt
+        robottelo.ui.locators:
+          as: locators
+  banner:
+    enabled: true
+    message: Welcome to {project_name} Interactive shell
+  init_script: |
```

Comment:
I am wondering if `manage` can support a flag or something to make this automatically.

I mean if I have something like the following on the config YAML file:

```yaml
shell:
  auto_import:
      objects:
        robottelo:
          as: robottelo
          import_submodules: true
```

That will make `robottelo` and all submodules of robottelo available as `robottelo.<submodule?` available.

Also the `import_submodules` can be a list telling what to import or using `*` as the first item to tell to import all available submodules. The list approach is more promising in my opinion.